README.md

```diff
@@ -91,10 +91,11 @@ nix eval --json 'github:kachick/dotfiles#homeConfigurations' --apply 'builtins.a
 
 ## Ubuntu
 
-1. Install [Nix](https://nixos.org/) package manager with [DeterminateSystems/nix-installer](https://github.com/DeterminateSystems/nix-installer) to enable [Flakes](https://nixos.wiki/wiki/Flakes) by default.
+1. Install [Nix](https://nixos.org/download/) and enable [Flakes](https://wiki.nixos.org/wiki/Flakes)
 
    ```bash
-   curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install --prefer-upstream-nix
+   sh <(curl --proto '=https' --tlsv1.2 -L https://nixos.org/nix/install) --daemon
+   sudo echo 'experimental-features = nix-command flakes' >> /etc/nix/nix.conf
```

Comment:
https://github.com/cachix/install-nix-action/blob/7ec16f2c061ab07b235a7245e06ed46fe9a1cab6/install-nix.sh#L56-L67
https://github.com/cachix/install-nix-action/blob/7ec16f2c061ab07b235a7245e06ed46fe9a1cab6/install-nix.sh#L27-L30
https://github.com/cachix/install-nix-action/blob/7ec16f2c061ab07b235a7245e06ed46fe9a1cab6/install-nix.sh#L69-L73
https://github.com/cachix/install-nix-action/blob/7ec16f2c061ab07b235a7245e06ed46fe9a1cab6/install-nix.sh#L82-L94


`--nix-extra-conf-file`

I didn't know this option 😮 

https://github.com/NixOS/nix/blob/f2253a00bc7778d3de836d59a9d52964003032e3/scripts/install-nix-from-tarball.sh#L87-L90
`https://github.com/NixOS/nix/pull/3623`

Reply:
https://github.com/NixOS/nix/blob/f2253a00bc7778d3de836d59a9d52964003032e3/scripts/install-multi-user.sh#L994-L1001

Is it available only for daemon(multi-user) installation? Should I avoid it in [container building](https://github.com/kachick/containers/issues/261)?